docs/fr/api/index.md

```diff
@@ -0,0 +1,2029 @@
+
+test('text', () => {
+  const wrapper = mount(Component);
+
+  expect(wrapper.find('p').text()).toBe('Bonjour tout le monde');
+});
+```
+
+### trigger
+
+Déclenche un évènement du DOM, par exemple `click`, `submit` ou `keyup`.
+
+**Signature :**
+
+```ts
+interface TriggerOptions {
+  code?: String
+  key?: String
+  keyCode?: Number
+  [custom: string]: any
+}
+
+trigger(eventString: string, options?: TriggerOptions | undefined): Promise<void>
+```
+
+**Utilisation :**
+
+`Component.vue`:
+
+```vue
+<template>
+  <span>Compteur: {{ count }}</span>
+  <button @click="count++">Cliquer ici</button>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      count: 0,
+    };
+  },
+};
+</script>
+```
+
+`Component.spec.js`:
+
+```js
+import { mount } from '@vue/test-utils';
+import Component from './Component.vue';
+
+test('trigger', async () => {
+  const wrapper = mount(Component);
+
+  await wrapper.find('button').trigger('click');
+
+  expect(wrapper.find('span').text()).toBe('Compteur: 1');
+});
+```
+
+Notez que `trigger` accepte un second argument pour transmettre des options à l'événement déclenché :
+
+```js
+await wrapper.trigger('keydown', { keyCode: 65 });
+```
+
+::: warning
+Vous devriez utiliser `await` lorsque vous appelez `trigger` pour vous assurer que Vue met à jour le DOM avant de faire une vérification.
+:::
+
+::: warning
+Certains événements, comme le fait de cliquer sur une case à cocher pour modifier son `v-model`, ne fonctionneront que si le test utilise `attachTo: document.body`.
+
+Sinon, l'événement `change` ne sera pas déclenché et la valeur de `v-model` ne changera pas en conséquence.
+:::
+
+### unmount
+
+Démonte l'application du DOM.
+
+**Signature :**
+
+```ts
+unmount(): void
+```
+
+**Utilisation :**
+
+Il ne fonctionne que sur le `VueWrapper` racine renvoyé par `mount`. Utile pour un nettoyage manuel après les tests.
+
+`Component.vue`:
+
+```vue
+<script>
+export default {
+  unmounted() {
+    console.log('démonté !');
+  },
+};
+</script>
+```
+
+`Component.spec.js`:
+
+```js
+import { mount } from '@vue/test-utils';
+import Component from './Component.vue';
+
+test('unmount', () => {
+  const wrapper = mount(Component);
+
+  wrapper.unmount();
+  // Le composant est supprimé du DOM.
+  // console.log a été appelé avec 'démonté !'
+});
+```
+
+## Propriétés de Wrapper
+
+### vm
+
+**Signature :**
+
+```ts
+vm: ComponentPublicInstance
+```
+
+**Utilisation :**
+
+L'instance d'application `Vue`. Vous pouvez accéder à toutes les [méthodes d'instance](https://v3.vuejs.org/api/instance-methods.html) et [propriétés d'instance](https://v3.vuejs.org/api/instance-properties.html).
+
+Notez que `vm` n'est disponible que sur un `VueWrapper`.
+
+:::tip
+En règle générale, testez les effets d'une `prop` transmise (une mise à jour du DOM, un événement émis, etc.). Cela rendra les tests plus puissants que simplement de faire une vérification sur le passage d'une prop.
+:::
```

Comment:
C'est traduit de ce passage :
:::tip
As a rule of thumb, test against the effects of a passed prop (a DOM update, an emitted event, and so on). This will make tests more powerful than simply asserting that a prop is passed.
:::